src/lib/date.ts

```diff
@@ -1,8 +1,32 @@
 import moment from "moment"
 
+const formatDateStatusDisplayDate = date => {
+  const momentToUse = moment.utc(date, "YYYY-MM-DD[T]HH:mm:ss")
+  const momentDate = momentToUse.format("MMM D")
+  const momentHour = momentToUse.format("ha")
+  if (!!momentHour && !!momentDate) {
+    return ` ${momentDate} at ${momentHour}`
+  } else if (!!momentDate) {
+    return ` ${momentDate}`
+  }
+}
+
+export function dateStatusDisplay(startAt, endAt) {
+  const thisMoment = moment()
+  const startMoment = moment(startAt, "YYYY-MM-DD[T]HH:mm:ss")
+  const endMoment = moment(endAt, "YYYY-MM-DD[T]HH:mm:ss")
+  if (thisMoment.isBefore(startMoment)) {
+    return `Opens${formatDateStatusDisplayDate(startAt)}`
+  } else if (thisMoment.isBefore(endMoment)) {
+    return `Closes${formatDateStatusDisplayDate(endAt)}`
+  } else {
+    return "Closed"
+  }
+}
+
 export function exhibitionPeriod(startAt, endAt) {
-  const startMoment = moment(startAt)
-  const endMoment = moment(endAt)
+  const startMoment = moment.utc(startAt, "YYYY-MM-DD[T]HH:mm:ss")
+  const endMoment = moment.utc(endAt, "YYYY-MM-DD[T]HH:mm:ss")
```

Comment:
Is `exhibitionPeriod` not used elsewhere that _shouldn’t_ change?

Reply:
My test was failing because I didn't include the moment format, so I thought I'd add it here too because I was the original author of this function. That being said I can remove but I believe moving forward moment.js wants the format indicated.

Reply:
Ok, that’s fine then, no need to change if you think that’s good 👍

Reply:
Hmm, this didn't sound right to me, so I did some fiddling with the code here. 

There is an _extremely_ devilish gotcha in this code — many of the fixture dates in the spec file are using en dashes to delimit the date parts instead of hyphens.

So on [L484](https://github.com/artsy/metaphysics/pull/1649/files?file-filters%5B%5D=.js&file-filters%5B%5D=.ts#diff-5ee543dceb7f3086afc41b9ce10c1be2R484) we've got:

```
          start_at: "2019–02–06T12:00:56+00:00",
```

instead of 

```
          start_at: "2019-02-06T12:00:56+00:00",
```
(At the risk of stating the obvious, en dashes are appropriate for _display_ but would never be returned as part of an ISO8601 date string.)

I think if you fix that, you will take care of the unexpected spec failure as well.

Reply:
Oh wow this was my issue.

Reply:
Fixed